index.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    
+
+    <title>mrcbq portfolio</title>
 </head>
+
 <body>
-    
+    <nav class="navbar">
+        <div class="navbar-left">
+            <p class="logo-text">My Logo</p>
+        </div>
+        <div class="navbar-right">
+            <img src="./icons/Icon - Menu.png" alt="burguer menu">
+        </div>
+    </nav>
+
+    <section class="main-container">
+        <div class="main-mobile">
+            <p class="title-orange">Hey There. <br> I'm Marco.</p>
+            <h2 class="role">I'm a Software Developer</h2>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="social-media">
+                <img src="./icons/Icon -GitHub.png" alt="Github Icon">
+                <img src="./icons/Icon - Linkedin.png" alt="Linkedin Icon">
+                <img src="./icons/Icon - Angelist.png" alt="Angelist Icon">
+                <img src="./icons/Icon - Twitter.png" alt="Twitter Icon">
+                <img src="./icons/Icon - Medium.png" alt="Medium Icon">
+            </div>
```

Comment:
- [ ] I think it would be better also for these list of icons to use a ```ul``` tag to group them, and ```li``` per icon; to explicitly mean a list of icons and also for the best HTML semantics 😎😎 with and ```a``` tag on each 👍 

_Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href._

Reply:
Great advice, thanks, It's done.